sqlmesh/__init__.py

```diff
@@ -112,10 +113,15 @@ def format(self, record: logging.LogRecord) -> str:
         return formatter.format(record)
 
 
-def enable_logging(level: int = logging.INFO) -> None:
+def debug_mode_enabled() -> bool:
```

Comment:
add to docs

Reply:
what would be a good place?

Reply:
I can add a section here https://sqlmesh.readthedocs.io/en/stable/reference/configuration/

Reply:
added